hub/saml-engine/src/integration-test/java/uk/gov/ida/integrationtest/hub/samlengine/apprule/CountryAuthnResponseTranslatorResourceTest.java

```diff
@@ -135,22 +207,28 @@ private void assertEquals(Assertion first, Assertion second) {
     }
 
     private SamlAuthnResponseTranslatorDto createAuthnResponseSignedByKeyPair(String publicKey, String privateKey) throws Exception {
+        return createAuthnResponseSignedByKeyPair(publicKey, privateKey, EncryptionConstants.ALGO_ID_BLOCKCIPHER_AES256_GCM, EncryptionConstants.ALGO_ID_KEYTRANSPORT_RSAOAEP);
```

Comment:
I'm going to be pedantic... 

This means we are only testing responses of AES256_GCM but we are accepting AES128_GCM as well (I can see it in the module class.)

Should we try to cover both cases?

Reply:
Okay I was being daft. Thx for pointing it out in person.